core/src/main/java/org/elasticsearch/rest/action/support/RestActions.java

```diff
@@ -64,18 +73,24 @@ public static long parseVersion(RestRequest request, long defaultVersion) {
     }
 
     static final class Fields {
+        static final String _NODES = "_nodes";
```

Comment:
I'm not following you? The bytes for the string have to sit somewhere and they always have to be there. When the compiler needs to use a string from the constant pool, it just emits a special instruction `ldc` to load a reference to the string onto the stack.